tdms/include/field.h

```diff
@@ -2,14 +2,44 @@
 #include "simulation_parameters.h"
 
 
+/**
+ * A generic grid entity. For example:
+ *
+ *      _____________________ (2, 1)
+ *     /           /         /
+ *    /                     /
+ *   /___________/_________/
+ *  (0, 0)     (1, 0)     (2, 0)
+ *
+ *  has I_tot = 2, J_tot = 1, K_tot = 0.
+ */
+class Grid{
+
+protected:
+    int I_tot = 0;
```

Comment:
Any reason not to go lower case on ijk here? I'm worried that `I_tot` could be interpreted as total electric current.